### PR TITLE
Column view width

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: src/view/components/cell/cell.handler.js [hunk 1]
```diff
@@ -14,26 +14,49 @@ class CellHandler extends Component {
...
+
+		// When navigate first or when animation wasn't applied we need to omit
+		// next navigation event to make handler to correct position.
+		let isValid = false;
+		model.navigationChanged.watch(e => {
+			if (e.hasChanges('cell')) {
+				const domCell = table.body.cell(e.state.rowIndex, e.state.columnIndex);
+				const cell = e.state.cell;
+				if (cell) {
+					const oldColumn = (e.changes.cell.oldValue || {}).column || {};
```
Comment (etokareva) [inline src/view/components/cell/cell.handler.js]: Confused a bit with these lines

Comment (klumba12) [reply src/view/components/cell/cell.handler.js]: oldValue can be null so to get something not nullable to oldColumn we write like that

Comment (klumba12) [reply src/view/components/cell/cell.handler.js]: any other ideas?

Comment (etokareva) [reply src/view/components/cell/cell.handler.js]: @klumba12 Maybe 
`` e.changes.cell.oldValue ? e.changes.cell.oldValue.column : {} ``
or strict null check in the first place

**Review by titovmx — APPROVED** [review]

**Pull request merged**